javascript/ql/src/semmle/javascript/frameworks/Request.qll

```diff
@@ -44,5 +44,13 @@ module Request {
     }
 
   }
+  
+  // using 'request' library to make http 'POST' and 'PUT' requests with message body. 
+  private class RequestPostBody extends HTTP::RequestBody {
+      RequestPostBody () {
+          this = DataFlow::moduleImport("request").getAMemberCall("post").getArgument(1) or
```

Comment:
I'd suggest you use `DataFlow::moduleMember("request", "post").getACall()` instead to also handle ES2015-style destructuring imports like this:

```
import { post } from "request";
post(...)
```